ChangeLog

```diff
@@ -1,3 +1,116 @@
+	  redbaron4. #3394
+	* UBI plugin: The new "ubifs" plugin reports block state for flash
+	  memory devices with UBIFS filesystem has been added. Thanks to pticon.
+	  #3204
+	* Varnish plugin: Support for Varnish 6 has been added, deprecated for
+	  ver 2, 3 and 5. Thanks to ismaelpuerto. #3360
+	  This has been reverted. Thanks to Matthias Runge #3414
+	* Write_HTTP plugin: Ability to configure cURL statistics has been
+	  added. Thanks to n2yen. #3265
+	* Write_Influxdb_UDP plugin: New plugin to send values to InfluxDB using
+	  line protocol via udp has been added. Thanks to carlospeon. #3162
+	* Zookeeper plugin: 'quorum' stat has been added. Thanks to chris9871.
+	  #2991
+	* collectd: Missing package in specfile has been fixed. Thanks to
+	  faxm0dem. #3399
+	* collectd: Factored out read_text_file_contents for reading text files
+	  and used it to fix a potential unterminated string in the thermal
+	  plugin. Thanks to igorpeshansky. #3359
+	* collectd: Error/warning messages has been extended with more
+	  information. Thanks to dago. #3386
+	* collectd: Regression caused by #3217 has been fixed. Thanks to
+	  rpv-tomsk. #3349
+	* build: version-gen.sh has been cleaned up. Thanks to wrp. #2635
+	* build: Linking with Python 3.8 has been fixed. Thanks to
+	  Self-Perfection. #3344
+	* build: Avoid $< (implied source) in non-inference rules / Building
+	  grpc with non-GNU make has been fixed. Thanks to efuss, dago. #3376,
+	  #3330
+	* filter_chain: Constant for max plugin name length introduced in #1034
+	  has been fixed. Thanks to dago. #3385
+	* ignorelist: Coding style has been fixed. Thanks to dago. #3397
+	* match_regex: Accept all data types of meta data. Thanks to ashie.
+	  #3231
+	* perl unixsock: Made the module robust against socket errors and
+	  disconnects. Thanks to guillemj. #3083
+	* tree-wide: Race condition when setting thread names has been fixed.
+	  Thanks to mkauf. #2762
+	* types.db: ping_droprate range spec has been fixed. Thanks to yousong.
+	  #3244
+	* utils/format_graphite: Comma sign has been added to the list of
+	  prohibited characters. Thanks to salemgolemugoo. #2998
+	* Aggregation plugin: Unreachable clause where both tmp_plugin and
+	  tmp_plugin_instance are non-empty has been fixed. Thanks to
+	  qingling128. #3350
+	* AMQP1 plugin: Leaks on error paths has been fixed. Thanks to
+	  ryan-mccabe. #3352
+	* Apache plugin: BusyWorkers/IdleWorkers update has been fixed. Thanks
+	  to zuckschwerdt. #3131
+	* CPU plugin, Memory plugin, Swap plugin, UUID plugin: Contain use of
+	  sysctl / sysctrlbyname to BSD variants, as deprecated on Linux. Thanks
+	  to zebity. #3342
+	* Exec plugin: Select has been replaced with poll to allow more then
+	  1024 open file descriptors. Thanks to kwiatrox. #3363
+	* GPU_Nvidia plugin: Build issues has been fixed. Thanks to rubenk.
+	  #3393
+	* Interface plugin: Reorder so that Linux-specific code is actually
+	  executed. Thanks to dago. #3378
```

Comment:
Previous Changelog records contains not account-names, but real Name and Surname.

We are not robots. We are people.

Reply:
You are absolutely right Pavel, have made that initially as was faster. Let me fix that (on wiki).

Reply:
What means "on wiki"?

This content (this file) needs to be updated.

Reply:
Apologies for delay, got pulled away.
By "on wiki" I ment [Version_5.11](https://collectd.org/wiki/index.php/Version_5.11) (should be up-to-date now, meaning got names where was able to find) which is then usually taken as input to [wiki2changelog.pl](https://collectd.org/wiki/index.php/Release_process#wiki2changelog.pl) to generate content for patches like this one.

Reply:
Ok, then closing this PR.